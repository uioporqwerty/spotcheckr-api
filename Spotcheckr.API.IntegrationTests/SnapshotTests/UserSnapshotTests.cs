using System.Threading.Tasks;
using HotChocolate;
using HotChocolate.Execution;
using Snapshooter.Xunit;
using Xunit;

namespace Spotcheckr.API.IntegrationTests.SnapshotTests
{
    public class UserSnapshotTests : BaseSnapshotTest
    {
	    [Fact]
	    public async Task GetAthlete_WithId_ReturnsAthlete()
	    {
		    var executor = await GetRequestExecutor();

		    var result = await executor.ExecuteAsync(new QueryRequest(new QuerySourceText(@"
query {
	athlete(id: 1) {
		id
	}
}
													")));
		    Snapshot.Match(result.ToJson());
	    }
    }
}
