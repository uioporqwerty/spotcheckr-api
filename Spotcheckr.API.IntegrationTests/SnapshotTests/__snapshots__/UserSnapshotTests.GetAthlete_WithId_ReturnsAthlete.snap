{
  "errors": [
    {
      "message": "Unexpected Execution Error",
      "locations": [
        {
          "line": 3,
          "column": 2
        }
      ],
      "path": [
        "athlete"
      ]
    }
  ]
}
